src/System.Linq.Expressions/src/System/Linq/Expressions/ConstantExpression.cs

```diff
@@ -12,11 +11,9 @@ namespace System.Linq.Expressions
     /// <summary>
     /// Represents an expression that has a constant value.
     /// </summary>
-    [DebuggerTypeProxy(typeof(Expression.ConstantExpressionProxy))]
+    [DebuggerTypeProxy(typeof(ConstantExpressionProxy))]
     public class ConstantExpression : Expression
     {
-        // Possible optimization: we could have a Constant<T> subclass that
-        // stores the unboxed value.
```

Comment:
Removed this comment because such unboxing would only be saved with either a lot of special-casing for constant expressions, or by pushing such generic typing all the way through expressions. Otherwise it just multiplies the amount of types that would be produced (especially in AOT) just to delay the boxing until later.